public/angular/templates/tags/modal.html

```diff
@@ -27,6 +27,10 @@ <h5 class="modal-title">
             A color must be specified.
           </small>
         </div>
+        <div class="form-group">
+          <label for="tagIsCommentTag" translate>Does tag enable comments?</label>
```

Comment:
Might want to rephrase this to better explain what it does. Like "Does this tag enable comment acquisition on posts?" or something. Might be fine though.